### PR TITLE
Add Videos tab and associated menu listings with video content

### PR DESCRIPTION
**Pull request by zollillo**
### Resolves

This PR introduces a new `Videos` tab for listing available video tutorials.

### Proposed Changes

- adds new  tab for `Videos` content
- adds new _button-like_ component used as menu item which - on click - open a modal overlay for playing the video 
- adjusts styles

### Reason for Changes

Missing video tutorials that are readily accessible directly from the _Programmieren mit der Maus_ page.

### Browser Coverage
Check the OS/browser combinations tested (At least 2)

Mac
 * [x] Chrome 
 * [x] Firefox 
 * [x] Safari

### CODE REVIEW

File: src/components/video-button-modal/video-button-modal.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <>
+      <div
+        className={styles.projectWrapper}
+        role="button"
+        onClick={() => setShowModal(true)}
+      >
+        <span className={styles.title}>{title}</span>
+        <span className={styles.image}>
+          {image ? <img src={image} /> : <img src={defaultImage} />}
```
Comment (bonflintstone) [inline src/components/video-button-modal/video-button-modal.jsx]:
```suggestion
          <img src={image || defaultImage} />
```

**Review by bonflintstone — APPROVED** [review]
Did not try it in the browser, but the code all looks real good :+1:

File: src/components/video-button-modal/video-button-modal.jsx (hunk 1 above)
Comment (P3nny) [reply src/components/video-button-modal/video-button-modal.jsx]: I did test it in the browser, added some minor text changes and talked to Nadja about tiny styling changes.

**Comment by P3nny** [comment]
@jh0ker I added title pictures for the music videos. And Nadja added the Tab icon for Videos. We have to find out about the right location to store these assets.

**Review by P3nny — APPROVED** [review]

**Pull request merged**